### ISSUE TITLE
Not able to Set Message Handler

### ISSUE THREAD

**Issue by thewodros** [mobile-web-messaging/MQTTKit#26]
Hi, 
I am trying to use the MQTTKit framework in my Swift project. I was able to connect to my server and subscribe to a topic using the: "connectToHost:(NSString*)host completionHandler:" and "subscribe(NSString *)topic withCompletionHandler:" methods respectively; however, I was not able to 
set a message handler as explained in the github doc here: https://github.com/jmesnil/MQTTKit#subscribe-to-a-topic-and-receive-messages.

Basically I don't see the "setMessageHandler:" method in the library.

Thanks,
Teddy
